src/Helpers/Schedule/Event.php

```diff
@@ -21,7 +21,7 @@ class Event
      * @param string      $slug globally unique
      * @param string[]    $persons id => name
      * @param string|null $language two letter code
-     * @param string|null $recording license (and opt out in XML, null if not recorded, empty if no license defined)/
+     * @param string      $recording license (and opt out in XML, null if not recorded, empty if no license defined)/
```

Comment:
The method type should be fixed here instead

Reply:
don't understand what you mean

Reply:
@ximex what i meant here is that the `protected string $recording = '',` should be `protected ?string $recording = '',` instead as it is supposed to be nullable by spec.

Reply:
done